ide-migration/server/migration/api/migration-service.mjs

```diff
@@ -108,7 +108,17 @@ export class MigrationService {
     }
 
     _buildHDIContainerName(duName, projectName) {
-        return `${duName}_${projectName}`.toUpperCase();
+        const groupAndContainerNamePattern = /[^\w_]/g;
+        const forbiddenCharactersReplacer = "_";
+        const forbiddenLeadingCharacter = "_";
+
+        let nameAccordingToConvention = `${duName}_${projectName}`.toUpperCase().replace(groupAndContainerNamePattern, forbiddenCharactersReplacer);
+
+        while (nameAccordingToConvention.substring(0, 1) == forbiddenLeadingCharacter) {
+            nameAccordingToConvention = nameAccordingToConvention.substring(1);
+        }
+
+        return nameAccordingToConvention.length ? nameAccordingToConvention : "NAME_CONVERSION_LEADS_TO_TRIM";
```

Comment:
I don't think it's a good idea to return a hardcoded string but I suppose this would not be needed if we take the approach I mentioned in the comment above

Reply:
OK. Fixed because of the "hdi_" prefix